data/data/bootstrap/openstack/files/etc/dhcp/dhclient.conf

```diff
@@ -1,2 +1 @@
 send dhcp-client-identifier = hardware;
```

Comment:
Not entirely sure what's the NM equivalent of this

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1661165